src/main/resources/static/javascript/mainApp/general.controller.js

```diff
@@ -37,6 +37,7 @@
                 uid: res.data.username,
                 uhUuid: res.data.uhUuid
             };
+            $scope.feedbackEmail = $scope.currentUser.uid + Message.Csv.EMAIL_SUFFIX;
```

Comment:
Would this line work if it was placed to where `$scope.feedbackEmail` is defined in `feedback.controller.js`?

Reply:
It doesn't work, I put a comment where $scope.feedbackEmail is defined but I'm not sure if that's the best approach

Reply:
I just tried it and it works:
![image](https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/assets/42422209/64f833ba-ab67-42d8-8663-40b1c8b3c01c)
![image](https://github.com/uhawaii-system-its-ti-iam/uh-groupings-ui/assets/42422209/65665234-6b37-43d0-8298-733557b4ce11)

Reply:
On my end, if you define it there, and then try and put an invalid email it will send anyway

Reply:
Oh ok, it's fine then.